lib/src/Uncertainty/Algorithm/Sensitivity/openturns/HSICEstimatorConditionalSensitivity.hxx

```diff
@@ -45,6 +45,7 @@ class OT_API HSICEstimatorConditionalSensitivity
 public:
 
   typedef Collection<CovarianceModel> CovarianceModelCollection;
+  typedef Collection <CovarianceMatrix>  CovarianceMatrixCollection;
```

Comment:
C++ question: is it necessary to typedef something in a child class if the typedef is already done in the parent class?